uvcgan/models/generator/__init__.py

```diff
@@ -0,0 +1,27 @@
+from uvcgan.base.networks    import select_base_generator
+from uvcgan.base.weight_init import init_weights
+from uvcgan.torch.funcs      import prepare_model
+
+from .vit       import ViTGenerator
```

Comment:
@usert5432 should we remove the ViTGenerator (and the `vit.py`)? (to prevent people using it as UVCGAN)

Reply:
I have been debating it with myself. On the one hand, yes. But, we still provide basic ResNet and UNet generators (inherited from the original CycleGAN). People still can select those models. So, the removal of `ViTGenerator` is insufficient to prevent people from using the wrong generators.

On the other hand, maybe keeping ViTGenerator will allow people to see how much better our hybrid architecture, compared to the vanilla ViT.

So, I will leave this decision up to you.